app/src/main/java/org/mozilla/focus/web/Download.java

```diff
@@ -34,17 +36,20 @@ public Download createFromParcel(Parcel source) {
     private final String mimeType;
     private final long contentLength;
     private final String userAgent;
+    private final boolean startFromContextMenu;
 
     public Download(@NonNull String url,
                     @NonNull String userAgent,
                     @NonNull String contentDisposition,
                     @NonNull String mimeType,
-                    long contentLength) {
+                    long contentLength,
+                    @NonNull boolean startFromContextMenu) {
```

Comment:
I can be wrong but it does not make sense to annotate a primitive type with NonNull to me.